src/client/Components/Pages/Home.jsx

```diff
@@ -1,36 +1,60 @@
+// @flow
+
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import Paper from '@material-ui/core/Paper';
 
 import NavBar from '../Util/NavBar';
+import validateAuth from '../../util/Util';
```

Comment:
Two `util` packages? Confusing.

Reply:
Poor naming on my part. Renamed it to `index.js` to remove the need for the extra `Util`